### ISSUE TITLE
[new-unit] Density property 

### ISSUE THREAD

**Issue by balporig** [ferraridamiano/ConverterNOW#156]
**Description**

Density property 

**Units to be added**

| Unit name                 | Property | Reference to the conversion formula                          |
| ------------------------- | -------- | ------------------------------------------------------------ |
| Gram/Cubic Centimeter     | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Gram/Cubic Meter          | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Gram/Cup                  | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Gram/Liter                | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| kilogram/Cubic Centimeter | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| kilogram/Cubic Meter      | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| kilogram/Liter            | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Ounce/Cubic Foot          | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Ounce/Cubic Yard          | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Ounce/Cubic Inch          | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Ounce/Gallon (UK)         | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Ounce/Gallon (US)         | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Pound/Cubic Foot          | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Pound/Cubic Inch          | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Pound/Cubic Yard          | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Pound/Gallon (US)         | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| PSI/1000 Feet             | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Ton/Cubic Yard (Short)    | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Ton/Cubic Yard (Long)     | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |
| Pound/Gallon (UK)         | Density  | [Density - Wikipedia](https://en.wikipedia.org/wiki/Density) |

**Comment by Kheirlb** [comment]
Hi @ferraridamiano, I see density is supported in https://github.com/ferraridamiano/units_converter/blob/main/lib/properties/density.dart, but now in the ConverterNOW app. Any reason why?

**Comment by Kheirlb** [comment]
Well, I tried adding the basics myself and it looks like there is just a decent chunk of work for making it user friendly with icons / text / etc.

![Image](https://github.com/user-attachments/assets/c13fb1bd-591c-44f8-a7a6-ac9862144483)

Additionally the big thing I would like lb/in³ which is not available just yet. I'll see if I can add that to units_converter.

**Comment by ferraridamiano** [comment]
> Hi @ferraridamiano, I see density is supported in https://github.com/ferraridamiano/units_converter/blob/main/lib/properties/density.dart, but now in the ConverterNOW app. Any reason why?

Nope, the only reason is the lack of time. I also prioritiezed other issues which were more requested.

I am very happy that someone else is working on this. When you are ready, please submit a PR

I like the icon!
